docs/apache-airflow-providers-apprise/connections.rst

```diff
@@ -0,0 +1,43 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/connection:apprise:
+
+Apprise Connection
+=======================
+
+The Apprise connection type enables connection to multiple services to send notifications. The complete list of services supported can be found  on the `Apprise Wiki <https://github.com/caronc/apprise/wiki#notification-services>`_.
+
+Default Connection IDs
+----------------------
+
+Apprise hooks point to ``apprise_default`` by default.
```

Comment:
We dont have such connection set in db/utils

We probably need a seperated PR to core to add it.

Reply:
I'll address that post we merge this PR :)

Reply:
We probably need however some doc notice about it because users of Airflow 2.4.0 for example won't have this default connection nor they will find apprise in the drop list..
so we probably need to explain how to setup the connection for them (using generic_connection?)

Reply:
@eladkal I'm sorry I didn't understand. 

If someone is using airflow - 2.4.0 
1. Without Apprise providers, generic connections won't be useful because the provider code is not there to understand what needs to be done with the generic connection
2. With Apprise provide you will have a connection type, which you can directly use therefore no need for a generic connection.

I'm sure I missed something here?

Reply:
Just to clarify: It won't be found in the list of default connections which are created with "init" (which we actually discourage. And the drop-down will be visiable when provider is installed (new connection types are provided by the provider being installed not by default connection created by initi).

I am not sure if we need to do that in Apprise docs, but maybe a separate documentation in general on how those things work and how default_* connections are created would be more useful ? 

Just to clarify behaviours and what to expect.

Reply:
Ya, I think a general document would be better, will also cater to all providers as well. I can raise a PR for the same.